redwood-flex-container/src/commonMain/kotlin/app/cash/redwood/flexbox/FlexContainer.kt

```diff
@@ -81,13 +80,90 @@ public class FlexContainer {
    */
   public val items: MutableList<FlexItem> = mutableListOf()
 
+  //region measure
+
+  /**
+   * Measures the flexbox's items according to the parent's [widthSpec] and [heightSpec] and returns
+   * the expected size of the flexbox.
```

Comment:
Minor but I think we should stick to using "container" here instead of "flexbox"